cartography/intel/aws/ec2/load_balancers.py

```diff
@@ -24,6 +24,9 @@ def get_loadbalancer_data(boto3_session: boto3.session.Session, region: str) ->
         elbs.extend(page['LoadBalancerDescriptions'])
     for elb in elbs:
         elb['region'] = region
+        elb['isPublicFacing'] = False
+        if not elb.get('VpcId'):
```

Comment:
If i understand correctly, if a load balancer is not connected to a VPC, it's public facing. right?

Can you please share any documentation or blog which you referred to implement this logic?

Reply:
Thanks for checking this. this is wrong logic. this logic about Elasticsearch but i have implemented in wrong file. i will correct it 
https://github.com/turbot/steampipe-mod-aws-perimeter/blob/099d6d9ea61afd4ab9f5aee116220f0f59466983/perimeter/network_access.sp#L99